osu.Game.Rulesets.Mania/Configuration/ManiaRulesetConfigManager.cs

```diff
@@ -15,32 +15,51 @@ public class ManiaRulesetConfigManager : RulesetConfigManager<ManiaRulesetSettin
         public ManiaRulesetConfigManager(SettingsStore? settings, RulesetInfo ruleset, int? variant = null)
             : base(settings, ruleset, variant)
         {
+            migrate();
         }
 
         protected override void InitialiseDefaults()
         {
             base.InitialiseDefaults();
 
-            SetDefault(ManiaRulesetSetting.ScrollTime, 1500.0, DrawableManiaRuleset.MIN_TIME_RANGE, DrawableManiaRuleset.MAX_TIME_RANGE, 5);
+#pragma warning disable CS0618
+            // Although obsolete, this is still required to populate the bindable from the database in case migration is required.
+            SetDefault<double?>(ManiaRulesetSetting.ScrollTime, null);
```

Comment:
I probably would have just gone with `-1` for the post-migrated value to avoid dealing with null, but all good.